rust/flexbuffers/src/reader/map.rs

```diff
@@ -46,30 +68,35 @@ impl<'de> std::fmt::Debug for MapReader<'de> {
     }
 }
 
-impl<'de> MapReader<'de> {
+impl<B: Buffer> MapReader<B> {
     /// Returns the number of key/value pairs are in the map.
     pub fn len(&self) -> usize {
+        let thing = self.clone();
```

Comment:
@CasperN Just noticed I left this debug code here which will have a performance impact, should I open up another PR or just have this PR reopened / edited?

Reply:
lol oops. I think you should open an new PR